cassandra-analytics-core/build.gradle

```diff
@@ -25,6 +25,8 @@ plugins {
     id('maven-publish')
 }
 
+apply from: "$rootDir/gradle/common/publishing.gradle"
```

Comment:
the `java` section below should be removed as it's already added in the publishing.gradle file